deltacat/aws/s3u.py

```diff
@@ -227,8 +231,12 @@ def read_file(
     except ClientError as e:
         if e.response["Error"]["Code"] in TIMEOUT_ERROR_CODES:
             # Timeout error not caught by botocore
-            raise RetryableError(f"Retry table download from: {s3_url}") from e
-        raise NonRetryableError(f"Failed table download from: {s3_url}") from e
+            raise RetryableError(
```

Comment:
Rename `DownloadTableTimedOutError`.

Reply:
Should retryability be purely a property of an error (for example DownloadTableError is is_retryable) as opposed to its own subclass?